main/autogen/autogen.cc

```diff
@@ -20,6 +20,19 @@ Reference &ReferenceRef::data(ParsedFile &pf) {
     return pf.refs[_id];
 }
 
+string_view Definition::typeAsStringView() {
```

Comment:
Happy to ditch this and just duplicate this switch (which appears elsewhere)—or, alternatively, update that other callsite.

Reply:
stylistically I think it's reasonable to assume you can omit the "string view" part from the method name and let the type itself document that for you.

Reply:
I'm guessing @DarkDimius may have a cleaner solution though

Reply:
+1 to @ngroman 's suggestion about the name.
We also use `show` for user-visible printing. Also `type` isn't for type here.
`showDefinitionKind`?